src/release-plan.ts

```diff
@@ -62,12 +62,11 @@ export async function planRelease({
   releaseSpecification: ReleaseSpecification;
   today: Date;
 }): Promise<ReleasePlan> {
-  const newReleaseName = today.toISOString().replace(/T.+$/u, '');
-  const newRootVersion = [
-    today.getUTCFullYear(),
-    today.getUTCMonth() + 1,
-    today.getUTCDate(),
-  ].join('.');
+  const newReleaseDate = formatDateAsISO(today, {
+    representation: 'date',
+  }).replace(/\D+/gu, '');
```

Comment:
Is this step meant to get rid of the dashes?

Reply:
I've just discovered that this can be done by setting the `format` option to `basic`

Reply:
Yeah, it's meant to get rid of the dashes. Thanks for the tip, I didn't look hard enough at the docs I guess 🤦🏻 Fixed in [a4f69eb](https://github.com/MetaMask/create-release-branch/pull/22/commits/a4f69eb6da28e0e34ae24d6156482e19cd232c37).